pyproject.toml

```diff
@@ -5,13 +5,17 @@ description = "A basic template for a discord bot with disnake."
 authors = ["OseSem"]
 license = "MIT"
 readme = "README.md"
+repository = "https://github.com/OseSem/bot-template"
+
+package-mode = false
 
 [tool.poetry.dependencies]
 python = "^3.10"
 disnake = "^2.9.1"
-disnake-ext-plugins = "^0.0.0a0"
+disnake-ext-plugins = {git = "https://github.com/disnakecommunity/disnake-ext-plugins", rev = "master"}
 coloredlogs = "^15.0.1"
 psutil = "^5.9.8"
+helply = {git = "https://github.com/dlchamp/helply", rev = "main"}
```

Comment:
Same as above, this should be pinned to a specific commit

Reply:
Both fixed.